common/src/main/java/uk/gov/justice/services/common/converter/Converter.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.services.common.converter;
+
+/**
+ * A converter converts a source object of type S to a target of type T.
+ */
+@FunctionalInterface
+public interface Converter<S, T> {
```

Comment:
We are effectively copying Spring's Converter interface :) http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/core/convert/converter/Converter.html